src/scripts/destinyTrackerApi/bulkFetcher.js

```diff
@@ -0,0 +1,81 @@
+import { itemListBuilder } from './itemListBuilder.js';
+
+class bulkFetcher {
```

Comment:
Without docs, a lot of the object roles are unclear. A sentence or two about what each of these objects are for and how they fit in with the others would be great.